project/src/Agents/GSTNode2DAgent.gd

```diff
@@ -0,0 +1,165 @@
+# A specialized steering agent that updates itself every frame so the user does
+# not have to.
+extends GSTNodeAgent
+class_name GSTNode2DAgent
+
+
+# The Node2D to keep track of
```

Comment:
In Godot, `body` generally refers to physics bodies

Reply:
My general question is: is it necessary or useful to support things other than physics bodies? As steered agents are moving entities, are there cases where you'd want agents that don't support collisions at all? At least with move and slide, they can slide against one another. And if there's no collision shape, they'll work similarly to node2Ds

Reply:
It would make things simpler, and if someone _is_ rolling their own movement mechanic without Godot's physics engine, then they'd probably have their own way of handling a steering behavior's acceleration anyway. So yeah, I think we can remove support for Node2D and just call for PhysicsBody and PhysicsBody2D respectively.